main.go

```diff
@@ -11,7 +11,10 @@ import (
 //go:generate go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs generate --provider-name clickhouse
 
 func main() {
-	providerserver.Serve(context.Background(), clickhouse.New, providerserver.ServeOpts{
+	err := providerserver.Serve(context.Background(), clickhouse.New, providerserver.ServeOpts{
 		Address: "clickhouse.cloud/terraform/clickhouse",
 	})
+	if err != nil {
+		panic(err)
```

Comment:
Maybe just print error and exit(1) ?
Not sure if panic is handled properly by code which executes the provider

Reply:
let me just ignore the error check. I checked some official provider code and people doesn't check that err